src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -809,6 +813,14 @@ export default class rd2EntryForm extends LightningElement {
         }
     }
 
+    /**
+     * @description Set the loading text to the specific text
+     * @param newLoadingText New loading text to be set to
+     */
+    changeLoadingTextTo(newLoadingText) {
```

Comment:
I think assigning directly to `this.loadingText` is still ok without this function.